packages/hoppscotch-common/src/components/app/Header.vue

```diff
@@ -231,30 +231,40 @@
       @invite-team="inviteTeam(editingTeamName, editingTeamID)"
       @refetch-teams="refetchTeams"
     />
+
+    <HoppSmartConfirmModal
+      :show="confirmRemove"
+      :title="t('confirm.remove_team')"
+      @hide-modal="confirmRemove = false"
+      @resolve="deleteTeam()"
```

Comment:
```suggestion
      @resolve="deleteTeam"
```

Isn't it better to specify just the function reference if no arguments are supplied? Also, aligns with the fact that the method is not immediately invoked.